oap-mail/src/main/java/oap/mail/SmtpTransport.java

```diff
@@ -21,7 +21,7 @@
 import java.util.regex.Pattern;
 
 @Slf4j
-public class SmtpTransport {
+public class SmtpTransport implements oap.mail.Transport {
```

Comment:
why fully qualified package?

Reply:
because we have import javax.mail.Transport; also here